biot_python_sdk/biot.py

```
@@ -266,7 +266,17 @@ def get_session_by_uuid(self, session_uuid):
         return self._make_authenticated_request(f"/device/v1/devices/usage-sessions?searchRequest={search_request_encoded}")
 
 
-
+    def get_ge_by_id(self, ge_id):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_ge_by_id`

(ANN201)